gef-ui/server/types/bank.ts

```diff
@@ -0,0 +1,18 @@
+import { BankReportPeriodSchedule } from '@ukef/dtfs2-common';
+
+export type Bank = {
+  id: string;
+  name: string;
+  mga: string[];
+  emails: string[];
+  companiesHouseNo: string;
+  partyUrn: string;
+  hasGefAccessOnly: boolean;
+  paymentOfficerTeam: {
+    teamName: string;
+    emails: string[];
+  };
+  utilisationReportPeriodSchedule: BankReportPeriodSchedule;
+  isVisibleInTfmUtilisationReports: boolean;
+  _id: string;
```

Comment:
i'd move this to the top so is clear there's two id's (bonus points if you can comment the difference)

Reply:
I'd comment the difference If I knew why there were two